### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
Created Product listing Page for e-Commerce web-App

please have a look , and give it a review

image
image
:
:
image

you can visit it live at :- https://epic-dijkstra-54a69e.netlify.app/

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
+.navigation-bar{
+    gap:60%;
+}
+
+body{
+    background: rgb(224, 220, 220);
```
Comment (someshtiwarii) [inline style.css:6]: try using color variables so you can reuse them

**Review by someshtiwarii — APPROVED** [review]
use color variables, other than that looks good

File: style.css (hunk 1 above)
Comment (Amandeep-sinha) [reply style.css:6]: Okay somesh will follow that thanku

**Review by percius47 — APPROVED** [review]
looks good

File: section1.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+                    <div class="inline"><h5>Rs. 799</h5><strike>Rs. 999</strike></div><a href="cart.html">Shop now</a> <a href="">Add To WishList</a>
+
+
+                </div>
+            
+
+            
+            </div>
+
+            <div class="card-div b">
```
Comment (pritam-kr) [inline section1.html:151]: what b is...you can write for a relatable name.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            <a href="https://www.linkedin.com/in/amandeep-sinha-1257251a2" target="_blank" class="social-icons"> <img
+                    src="https://cdn-icons-png.flaticon.com/512/61/61109.png" alt="icon"></a>
+            <a href="https://amandeepcl.netlify.app" class="social-icons" target="_blank"> <img
+                    src="https://cdn1.vectorstock.com/i/1000x1000/22/45/portfolio-icon-vector-2182245.jpg"
+                    alt="icon"></a>
+        </div>
+        Developed and Designed by : Amandeep<br><br>
+    </footer>
+</body>
+</html>
```
Comment (pritam-kr) [inline index.html:73]: remove red mark by breaking line

File: style.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+#hover-img{
+    width:95%;
+}
+
+.hero-text{
+    text-align: center;
+    padding:1rem;
+    padding-bottom: 0;
+    font-size:xx-large;
+}
```
Comment (pritam-kr) [inline style.css:27]: give space after the property name.

**Review by pritam-kr — APPROVED** [review]
Good to merge

**Pull request merged**